src/java/org/apache/cassandra/db/marshal/PartitionerDefinedOrder.java

```diff
@@ -130,6 +130,12 @@ public void validate(ByteBuffer bytes) throws MarshalException
         throw new IllegalStateException("You shouldn't be validating this.");
     }
 
+    @Override
+    public <V> boolean isNull(V buffer, ValueAccessor<V> accessor)
```

Comment:
NOTE: `text` and `byte[]` treat the empty set as non-null, so this class will convert those to `null`... I didn't want to really go down the rabbit hole to figure out the impact to this... but this isn't "correct" from the types point of view...